tests/system/providers/slack/conftest.py

```diff
@@ -14,3 +14,15 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+import pytest
+
+REQUIRED_ENV_VARS = (
+    "AIRFLOW_CONN_SLACK_DEFAULT",
+    "SYSTEM_TESTS_ENV_ID",
```

Comment:
System env id will be inherited from this line: https://github.com/apache/airflow/blob/9216489d9a25f56f7a55d032b0ebfc1bf0bf4a83/tests/system/conftest.py#L26 so there is no need to define it as required here.